### PR TITLE
[wip] add to sample, test and manifest

### PR DESCRIPTION
**Pull request by rmgogogo**
add myself to sample, test and manifest for better well know the changes for hosted

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2893)

### CODE REVIEW

File: samples/OWNERS [hunk 1]
```diff
@@ -3,9 +3,11 @@ approvers:
   - gaoning777
   - hongye-sun
   - numerology
+  - renming
```
Comment (numerology) [inline samples/OWNERS]: I guess you mean the following?
```suggestion
  - rmgogogo
```

**Comment by rmgogogo** [comment]
@googlebot I fixed it.

File: samples/OWNERS (hunk 1 above)
Comment (rmgogogo) [reply samples/OWNERS]: you are too quick, :-)

**Comment by numerology** [comment]
/lgtm
/approve
For the sample and test bits

**Comment by rmgogogo** [comment]
@googlebot I fixed it.

**Comment by rmgogogo** [comment]
@googlebot I fixed it.

**Pull request closed without merging**